src/components/Home/Home.tsx

```diff
@@ -42,24 +45,24 @@ const Home: React.FunctionComponent = () => {
 
   return (
     <main className={`${styles.main} ${styles[theme]}`}>
+      <HashNav />
       <Search />
-      {locationError && (
-        <ErrorComponent
-          message="Your current location is unavailable at the moment. Please check your browser settings or use search bar to search for location."
-          button="TRY_AGAIN"
-        />
-      )}
+      {locationError && !cityId && <ErrorComponent message={locationError} button="TRY_AGAIN" />}
       {userHasToken ? (
         <>
           <div className={styles.container}>
             <CurrentWeather location={locationParam} locationInfo={locationInfo} />
             <Today3Hourly location={locationParam} />
           </div>
+          <Today1Hourly location={locationParam} />
           <DailyForecast location={locationParam} />
         </>
-      ) : (
+      ) : locationError ? null : (
```

Comment:
why not like this? 
`locationError && !cityId ? () : userHasToken ? () :  ()`
`i'm just confused about `locationError ? null` line

Reply:
changed to your suggestion. It works as I wanted, thanks.